examples/12-fluids/00-fluids_model.py

```diff
@@ -0,0 +1,193 @@
+"""
+.. _ref_fluids_model:
+
+Explore Fluids models
+------------------------------------------------------
+
+"""
+
+###############################################################################
+# Exploring an Ansys Fluent model
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This example demonstrates how you can explore an Ansys Fluent model. Import
+# the result file and create a model
+
+import ansys.dpf.core as dpf
+from ansys.dpf.core import examples
+
+path = examples.download_fluent_axial_comp()["flprj"]
+ds = dpf.DataSources(path)
+model = dpf.Model(data_sources=ds)
+
+
+###############################################################################
+# Exploring the mesh
+# ------------------
+# Explore the mesh through the MeshInfo. The MeshInfo provides metadata information
+# about the mesh. For fluid models, it is useful to know the cell and face zones,
+# as well as the topological relationships between them. First get all the available
+# information in the MeshInfo
+
+minfo = model.metadata.mesh_info
+print(
+    "\n".join(
+        "{}\t{}".format(k, v)
+        for k, v in minfo.generic_data_container.get_property_description().items()
+    )
+)
+
+###############################################################################
+# Then, get the bodies and their names in the model with the "body_names" ``StringField``,
+# which provides a relationship between body IDs and names. In this model there are two
+# bodies.
+
+print(minfo.get_property("body_names"))
```

Comment:
@rafacanton you do not actually need to `print()`, as it will be rendered like a notebook, and the last thing in each cell is printed out anyway